var/spack/repos/builtin/packages/redis/package.py

```diff
@@ -18,16 +18,14 @@ class Redis(MakefilePackage):
     """
 
     homepage = "https://redis.io"
-    urls = [
-        "https://download.redis.io/releases/redis-7.0.5.tar.gz",
-        "https://github.com/redis/redis/archive/refs/tags/7.0.5.tar.gz",
-    ]
+    url = "https://github.com/redis/redis/archive/refs/tags/7.4.0.tar.gz"
     git = "https://github.com/redis/redis.git"
 
     maintainers("lpottier")
 
     license("BSD-3-Clause")
 
+    version("7.4.0", sha256="74effebe3d0e3ac582f5248492c13e7ca4918949b58b45873d37fc870a4b1fc6")
     version("7.0.5", sha256="67054cc37b58c125df93bd78000261ec0ef4436a26b40f38262c780e56315cc3")
```

Comment:
Can you mark the older versions affected by https://nvd.nist.gov/vuln/detail/CVE-2023-36824 as deprecated? That one only affects the 7.0 versions, but I imagine the most recent 6 version is also affected by at least one high or critical CVE (informal cutoff for deprecation).

Reply:
See eg https://github.com/spack/spack/pull/46873